### PR TITLE
feat(iac): #IAC-112 update example with iac command

### PR DESCRIPTION
**Pull request by carla-gitguardian**
Update `.gitguardian.example.yaml` with IaC command

### CODE REVIEW

File: .gitguardian.example.yml [hunk 1]
```diff
@@ -36,3 +36,15 @@ secret:
...
+  # Exclude files and paths by globbing
+  ignored-paths:
+    - 'tests/'
+
+  # Ignore IaC vulnerability types by their id
+  ignored-policies:
+    - GG_IAC_0000
+
+  # Precise vulnerabilities minimum severity to scan for
+  minimum-severity: LOW
```
Comment (sylvain-baud-gg) [inline .gitguardian.example.yml:50]: Maybe use another value for minimum-severity.

As LOW is the lowest possible minimum severity, this parameters doesn't change anything.

File: .gitguardian.example.yml [hunk 2]
```diff
@@ -36,3 +36,15 @@ secret:
   # Detectors to ignore.
   ignored-detectors: # default: []
     - Generic Password
+
+iac:
+  # Exclude files and paths by globbing
+  ignored-paths:
+    - 'tests/'
```
Comment (sylvain-baud-gg) [inline .gitguardian.example.yml:43]:
```suggestion
    - 'tests/*'
```

File: .gitguardian.example.yml [hunk 3]
```diff
@@ -36,3 +36,15 @@ secret:
   # Detectors to ignore.
   ignored-detectors: # default: []
     - Generic Password
+
+iac:
+  # Exclude files and paths by globbing
+  ignored-paths:
```
Comment (sylvain-baud-gg) [inline .gitguardian.example.yml:42]: Better to have an example with at least two ignored paths to show it is possible

File: .gitguardian.example.yml [hunk 4]
```diff
@@ -36,3 +36,15 @@ secret:
...
+    - 'tests/'
+
+  # Ignore IaC vulnerability types by their id
+  ignored-policies:
+    - GG_IAC_0000
```
Comment (sylvain-baud-gg) [inline .gitguardian.example.yml:47]: Better to have an example with at least two ignored policies to show it is possible

File: .gitguardian.example.yml [hunk 5]
```diff
@@ -36,3 +36,15 @@ secret:
...
+
+  # Precise vulnerabilities minimum severity to scan for
```
Comment (sylvain-baud-gg) [inline .gitguardian.example.yml:49]:
```suggestion
  # Minimum severity of the policies
```

File: .gitguardian.example.yml [hunk 6]
```diff
@@ -36,3 +36,15 @@ secret:
   # Detectors to ignore.
   ignored-detectors: # default: []
     - Generic Password
+
+iac:
+  # Exclude files and paths by globbing
+  ignored-paths:
+    - 'tests/'
+
+  # Ignore IaC vulnerability types by their id
```
Comment (sylvain-baud-gg) [inline .gitguardian.example.yml:45]:
```suggestion
  # IaC vulnerabilities to ignore.
```

**Review by sylvain-baud-gg — CHANGES_REQUESTED** [review]

File: .gitguardian.example.yml (hunk 2 above)
Comment (carla-gitguardian) [reply .gitguardian.example.yml:43]: Both work, only `test/` is recursive and `test/*` is not. Why do you think adding `*` is best?

Comment (sylvain-baud-gg) [reply .gitguardian.example.yml:43]: I think it's good to be consistent with the secret example.

File: .gitguardian.example.yml (hunk 1 above)
Comment (carla-gitguardian) [reply .gitguardian.example.yml:50]: This value was suggested by the linear ticket. As the other ones, maybe it is intended to not change anything ; such as having only one policy with id `GG_IAC_0000`?

Comment (sylvain-baud-gg) [reply .gitguardian.example.yml:50]: See Henri's answer on the Linear issue

**Review by GG-HH — APPROVED** [review]

**Review by sylvain-baud-gg — APPROVED** [review]

**Pull request merged**